s/features/chat/components/xiome-chat/renderers/render-chat-post.ts

```diff
@@ -22,7 +23,7 @@ export function renderChatPost({
 				${isModerator
 					? html`
 						<span class=moderation>
-							<xio-button title="mute" @press=${mute}>${muteIcon}</xio-button>
+							<xio-button title="mute" @press=${mute}>${mutedIds.includes(post.userId) === true ? null : muteIcon}</xio-button>
```

Comment:
- instead of making the button disappear, can we simply set it to disabled?
- or perhaps there is a different set of svg icons, and the svg icon can switch to something that implies that mute is in effect?